tests/util/translator/parse/elements/long-text/long-text-schema-parser.spec.tsx

```diff
@@ -0,0 +1,65 @@
+import { ITextareaSchema } from "@lifesg/web-frontend-engine/components/fields";
+import {
+    EElementType,
+    EValidationType,
+    IValidation,
+} from "src/context-providers";
+import { LongTextSchemaParser } from "src/translator/parse/elements";
+import { generateMockElementSchema } from "tests/util/translator/helper";
+
+describe("LongTextSchemaParser", () => {
+    beforeEach(() => {
+        global.ResizeObserver = jest.fn().mockImplementation(() => ({
+            observe: jest.fn(),
+            unobserve: jest.fn(),
+            disconnect: jest.fn(),
+        }));
+    });
+
+    afterEach(() => {
+        jest.restoreAllMocks();
+        jest.resetAllMocks();
+    });
+
+    it("should parse every validation type correctly if all present", () => {
+        const elementId = "mock123";
+        const mainLabel = "This is a label";
+        const subLabel = "This is a sub label";
+
+        const maxErrorMessage = "Max error message";
+
+        const MOCK_SCHEMA = generateMockElementSchema({
+            id: elementId,
+            label: {
+                mainLabel,
+                subLabel,
+            },
+            uiType: EElementType.TEXTAREA,
+            validation: [
+                {
+                    max: 10,
+                    errorMessage: maxErrorMessage,
+                },
+            ],
+        })[elementId] as ITextareaSchema;
+
+        const parsedSchema = LongTextSchemaParser.schemaToElement(
+            MOCK_SCHEMA,
+            elementId,
+            {}
+        );
+
+        const expectedParsedValidation: IValidation[] = [
+            {
+                validationType: EValidationType.MAX_LENGTH,
+                validationRule: "10",
+                validationErrorMessage: maxErrorMessage,
+            },
+        ];
```

Comment:
it will be clearer to split the tests for the validation parsing into separate test cases, this is especially so for the elements that can support multiple types of validation